packages/web3-shared/base/src/utils/timestamp.ts

```diff
@@ -0,0 +1,4 @@
+export function isValidTimesTamp(time?: number) {
+    if (!time) return
+    return new Date(time).toString() !== new Date(1).toString()
```

Comment:
After Ethereum's release date.
```suggestion
    return new Date(time).toString() !== new Date(1).toString()
```

Reply:
before 2010-1-1 is inValid